packages/react/src/components/baseCheckout/index.tsx

```diff
@@ -103,6 +103,30 @@ const BaseCheckout = ({
     searchParams
   );
 
+  const getIsBalanceEnough = useCallback(
+    (
+      customPrice?: number,
+      quantity?: number,
+      currency?: string,
+      interval?: number
+    ) =>
+      AvailableBalanceService.getIsBalanceEnough({
```

Comment:
boolean function naming is better with is or has prefixes. let's have isBalanceEnough here

Reply:
same as above